waggle-dance-core/pom.xml

```diff
@@ -178,6 +178,10 @@
           <groupId>org.eclipse.jetty</groupId>
           <artifactId>jetty-runner</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
This dependency is also pulled in by hive-common shouldn't it be excluded from here as well?

Reply:
Yes, I think you are right ,should exclude from hive-common as well.